lib/Reader/reader.cpp

```diff
@@ -336,7 +336,7 @@ uint32_t parseILOpcode(uint8_t *ILInput, uint32_t ILOffset, uint32_t ILSize,
   if (TheOpcode == ReaderBaseNS::CEE_ILLEGAL) {
     if (ReportErrors) {
       if (Reader == nullptr) {
-        ReaderBase::fatal(CORJIT_BADCODE);
+        dbPrint("parseILOpcode: Bad Opcode\n");
```

Comment:
No. But we do not want it do. Reader is only null when this method is called from
printMSIL. I don't want printMSIL to terminate execution if there is a problem.